app/apps/server/communication/uikit.js

```diff
@@ -14,6 +15,25 @@ const apiServer = express();
 
 apiServer.disable('x-powered-by');
 
+let corsEnabled = false;
+let allowListOrigins = [];
+
+settings.get('API_Enable_CORS', (_, value) => { corsEnabled = value; });
```

Comment:
While testing this, noticed some assets urls are affected by this setting, is that the expected behavior? 🤔

Reply:
Hey @KevLehman Could you share more info about this? I think, ideally it shouldn't affect any such assets url?

Reply:
Thanks for pointing this out! It seems this was an issue bcs I was applying the CORS filter on top of app the routers while actually we should just apply it on the `ui.interaction` endpoint. I've fixed the same now :+1:

Reply:
It works as expected now :)

Reply:
Great! Could you approve this as well :smile: